pom.xml

```diff
@@ -106,7 +106,13 @@
             <version>${elasticsearch.version}</version>
             <scope>test</scope>
         </dependency>
-    </dependencies>
+	<dependency>
+           <groupId>org.testng</groupId>
+           <artifactId>testng</artifactId>
+           <version>6.8.8</version>
+           <scope>test</scope>
+       </dependency>
+   	 </dependencies>
```

Comment:
Why is this change (and the two below) necessary?

Reply:
Hi @mpereira  please update

Reply:
Hi @mpereira https://github.com/Automattic/elasticsearch-statsd-plugin/pull/29 has merge to master please merge statsd_6.3.1 in mesosphere/elasticsearch-statsd-plugin master